File: MyFirstApplicationG31/app/src/main/java/com/example/myfirstapplicationg_31/MainActivity.java
package com.example.myfirstapplicationg_31;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.EditText;
import android.widget.ImageView;
import android.widget.TextView;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {
//    public void onClickMeButton(View view){
//        Log.i("INFO", "onClickMeButton");
//
//        TextView textViewAppTitle = (TextView) findViewById(R.id.textViewAppTitle);
//        EditText editTextPersonName = (EditText) findViewById(R.id.editTextPersonName);
//
//        Log.i("USER_INPUT", editTextPersonName.getText().toString());
//
//        textViewAppTitle.setText(editTextPersonName.getText().toString());
//
//        Toast.makeText(this, "Hi, " + editTextPersonName.getText().toString(), Toast.LENGTH_SHORT).show();
//    }

    public void onSwitchCat(View view){
        ImageView image = (ImageView) findViewById(R.id.imageView);
        image.setImageResource(R.drawable.cat_1);
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: Java_And_Kotlin/app/src/main/java/com/example/java_and_kotlin/MainActivity.kt
package com.example.java_and_kotlin

import android.content.Intent
import android.os.Bundle
import android.view.Menu
import android.view.MenuItem
import android.view.View
import android.widget.TextView
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    lateinit var userNameTextView: TextView
    lateinit var myUser: Person

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        userNameTextView = findViewById(R.id.userNameTextView)
        myUser = Person("John Smith", 45)
        userNameTextView.text = myUser.getInfo()

        // actionBar?.hide()
    }

    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
        menuInflater.inflate(R.menu.main_menu, menu)
        return true
        // return super.onCreateOptionsMenu(menu)
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {

        if( item.itemId == R.id.mProfile ) {
//            Toast.makeText(this, "Profile Clicked", Toast.LENGTH_LONG).show()
            val profileIntent = Intent(this, ProfileActivity::class.java);
            startActivity(profileIntent)
        }
        else if ( item.itemId == R.id.mLogOut ) {
            // Toast.makeText(this, "LogOut Clicked", Toast.LENGTH_LONG).show()

            val loginIntent = Intent(this, LoginActivity::class.java);
            startActivity(loginIntent)
            finish()
        }

        return super.onOptionsItemSelected(item)
    }

    fun onUpdateUserInfo(v: View){
        userNameTextView.text = myUser.getInfo()
    }
}